requests_toolbelt/__init__.py

```diff
@@ -27,7 +27,7 @@
 
 __all__ = [
     'GuessAuth', 'MultipartEncoder', 'MultipartEncoderMonitor',
-    'MultipartDecoder', 'SSLAdapter', 'SourceAddressAdapter',
+    'MultipartDecoder', 'SSLAdapter', 'SourceAddressAdapter', 'SNIAdapter',
```

Comment:
We've stopped adding things to the top level module. We plan to remove all of these in 1.0.0. I'll remove this when I merge it.